lighthouse-core/lib/statistics.js

```diff
@@ -28,37 +34,6 @@ function erf(x) {
   return sign * (1 - y * Math.exp(-x * x));
 }
 
-/**
```

Comment:
Not clear why this wasn't deleted in #10715. Possibility someone was using it? Hard to let go of our old friend the podr?

Searching github the only references I can find that aren't forks of this file are references to forks of this file from four year old forks of Lighthouse, so I think we're good to delete.

Farewell podr! You'll live on as an increasingly mystifying reference in our desmos graphs.